src/pages/CodeEditor/CodeEditor.sagas.ts

```diff
@@ -7,6 +7,7 @@ import {
   take,
   fork,
 } from "redux-saga/effects";
+import { Buffer } from "buffer";
```

Comment:
This is the only important line in the PR. This explicitly imports the buffer package @kapooramanpreet